home_work_6/index.js

```diff
@@ -0,0 +1,144 @@
+const checkIfValueNotValid = (variable) => {
+  // undefined == null -> true
```

Comment:
Considering that `undefined` and `null` are different primitives, I think  it's incorrect to use one equality operator for both

Reply:
ok